src/config.ts

```diff
@@ -0,0 +1,62 @@
+import { makeConfig } from 'cleaner-config'
+import { asArray, asNumber, asObject, asOptional, asString } from 'cleaners'
+
+export const asConfig = asObject({
+  couchUri: asOptional(asString, 'http://admin:password@localhost:5984'),
+  httpPort: asOptional(asNumber, 8008),
+  infoServerAddress: asOptional(asString, 'info1.edge.app'),
+  infoServerApiKey: asOptional(asString, 'hex'),
```

Comment:
Instead of `asOptional(asString, 'hex')`, maybe do `asOptional(asString, '') // hex`. This isn't JSON anymore, so comments are a thing.